internal/pkg/vm/actor/builtin/initactor/init.go

```diff
@@ -21,12 +24,26 @@ type Actor struct{}
 
 // State is the init actor's storage.
 type State struct {
-	Network string
+	Network    string
+	AddressMap cid.Cid `refmt:",omitempty"`
+	IdMap      cid.Cid `refmt:",omitempty"`
+	NextID     types.Uint64
+}
+
+// assignNewID returns the nextID and increments the counter
+func (s *State) assignNewID() types.Uint64 {
+	id := s.NextID
+	s.NextID++
+	return id
 }
 
 // Actor methods
 const (
-	GetNetwork types.MethodID = iota + 32
+	Exec                 types.MethodID = 3
+	GetActorIDForAddress types.MethodID = 4
```

Comment:
The [spec](https://filecoin-project.github.io/specs/#InitActor) only specifies ids for these two methods.